### PR TITLE
Second Java HomeWork

### PR DESCRIPTION
**Pull request by 1699kratos**
## ISSUE:
First java homework from https://www.hackerrank.com/domains/java
Second Java HomeWork from https://www.hackerrank.com/domains/java

## FIX:
*Added more polymorphism in the main class.
*Create a class for each excercise  
*Create mini project
*Solved Java Inheritance I, II, and Output Formatting 
*Use of inheritance, encapsulation, and polymorphism, taking as an example vehicle classes.
*New Attendance 
*Created a Main class for vehicles
*Return string eith methods 
## EVIDENCES:
image
image
image
image
image
image

### CODE REVIEW

File: JavaErickHernandez/Day2/JavaFormatting.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
+package bootcamp;
+
+import java.util.Scanner;
+
+public class JavaFormatting {
+	public static void Main(String[] args) {
+        Scanner sc=new Scanner(System.in);
+        System.out.println("================================");
```
Comment (edbermeo) [inline JavaErickHernandez/Day2/JavaFormatting.java:8]: Avoid the usage of magic strings

Magic Strings
https://deviq.com/antipatterns/magic-strings

File: JavaErickHernandez/Day2/JavaFormatting.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+        for(int i=0;i<3;i++){
+            String s1=sc.next();
+            int x=sc.nextInt();
+            System.out.printf("%-15s%03d\n", s1,x);
+        }
+        System.out.println("================================");
```
Comment (edbermeo) [inline JavaErickHernandez/Day2/JavaFormatting.java:14]: Avoid the usage of magic strings

Magic Strings
https://deviq.com/antipatterns/magic-strings

**Review by edbermeo — CHANGES_REQUESTED** [review]

**Review by luisalanmz — APPROVED** [review]
I liked your proposal to put evidence. Just answer the comments. You're doing well.

**Review by edbermeo — APPROVED** [review]
OK

**Pull request merged**